plugin.rb

```diff
@@ -108,12 +108,21 @@ def ensure_models
   end
 
   post '/IpamDriver.ReleaseAddress' do
-    Addresses::Release.run!(
-      pool_id: params['PoolID'],
-      address: params['Address']
-    )
-
-    json({})
+    begin
+      Addresses::Release.run!(
+        pool_id: params['PoolID'],
+        address: params['Address']
+      )
+
+      json({})
+    rescue Mutations::ValidationException => exc
+      if exc.errors['address'] && exc.errors['address'].symbolic == :zombie
+        status 202
```

Comment:
Actually just realized that cleanup is not "automatic", it's triggered by the agent. So probably 409 is more logical here.